nsdb-http/src/main/scala/io/radicalbit/nsdb/web/routes/QueryApi.scala

```diff
@@ -89,7 +90,7 @@ case class QueryBody(@(ApiModelProperty @field)(value = "database name ") db: St
                      @(ApiModelProperty @field)(value = "namespace name ") namespace: String,
                      @(ApiModelProperty @field)(value = "metric name ") metric: String,
                      @(ApiModelProperty @field)(value = "sql query string") queryString: String,
-                     @(ApiModelProperty @field)(value = "timestamp lower bound condition",
+                     @(ApiModelProperty @field)(value = "timestamp lower bound condition ",
```

Comment:
why space here?

Reply:
I see that some other values are with a final space and i think it was a typo but honestly i don't now if the typo is the space or without the space. Any suggestions @saveveltri ?

Reply:
there isn't any rule nor any best practice afaik. At a first sight, the space does not give any added value, unless it improves the swagger UI look&feel, but I don't think so. We can remove all the spaces